src/core/handlers/GraphQLHandler.ts

```diff
@@ -156,10 +161,8 @@ Consider naming this operation or using "graphql.operation()" request handler to
       return false
     }
 
-    const hasMatchingUrl = matchRequestUrl(
-      new URL(args.request.url),
-      this.endpoint,
-    )
+    const hasMatchingUrl = this.hasMatches(args)
```

Comment:
The diff makes this look like it's called in the same function body as the call above, but it is not

Reply:
Yeah, I noticed that but it's okay 👍

Reply:
modified this now, so `match` gets returned from parse, and passed to predicate.  We do this in the http handler as well, so it adds a bit of uniformity between those